src/EditorFeatures/Core.Wpf/Workspaces/WpfTextBufferVisibilityTracker.cs

```diff
@@ -172,6 +210,11 @@ private void UpdateTextViews(IEnumerable<ITextView> associatedTextViews)
             }
 
             private void VisualElement_IsVisibleChanged(object sender, System.Windows.DependencyPropertyChangedEventArgs e)
+            {
+                TriggerCallbacks();
+            }
+
+            internal void TriggerCallbacks()
```

Comment:
💡 This should be `private` with a `TestAccessor`.

Reply:
➡️ Missed that this was inside a nested private class. Current pattern is correct.

Reply:
Good to know. Thanks.